adobe_vipm/adobe/mixins/order.py

```diff
@@ -184,9 +185,10 @@ def create_preview_order(self, context: Context) -> dict | None:
         }
         self._process_new_lines(context, flex_discounts, payload)
         self._process_upsize_lines(context, flex_discounts, payload)
-        if context.deployment_id:
```

Comment:
I'd like to understand the change better. Why can't we store the deployment_id on the context but keep getting it dynamically here each time it is run? Couldn't we just set it at the beginning when we setup context? It is not going to change, is it?

Reply:
Maybe we can do it on next PR, today we need to fix production asap